Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3710,11 +3710,11 @@ def gen2021HiMix(fragment,howMuch):
 defaultDataSets['2017Design']='CMSSW_12_0_0_pre4-113X_mc2017_design_v5-v'
 defaultDataSets['2018']='CMSSW_12_0_0_pre4-113X_upgrade2018_realistic_v5-v'
 defaultDataSets['2018Design']='CMSSW_12_0_0_pre4-113X_upgrade2018_design_v5-v'
-defaultDataSets['2021']='CMSSW_12_0_0_pre4-120X_mcRun3_2021_realistic_v2-v'
-defaultDataSets['2021Design']='CMSSW_12_0_0_pre4-120X_mcRun3_2021_design_v2-v'
-defaultDataSets['2021FS']='CMSSW_12_4_0_pre4-124X_mcRun3_2021_realistic_v1_Run3FastSim_FastSim-v'
-defaultDataSets['2023']='CMSSW_12_0_0_pre4-120X_mcRun3_2023_realistic_v2-v'
-defaultDataSets['2024']='CMSSW_12_0_0_pre4-120X_mcRun3_2024_realistic_v2-v'
+defaultDataSets['2021']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_realistic_v10_BS2022-v'
+defaultDataSets['2021Design']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_design_v7_BS2022-v'
+defaultDataSets['2021FS']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_realistic_v10_BS2022_FastSim-v'
+defaultDataSets['2023']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_realistic_v10_BS2022-v'
```

Comment:
Just my curiosity, why do we end up with 2022 for all 2023 and 2024?
@kskovpen @cms-sw/alca-l2

Reply:
Hi @srimanob , thanks for your comment. @cms-sw/alca-l2 do you know if there are any differences in GT configurations among these years? If there are, we can additionally produce these samples.

Reply:
Not at the GEN-SIM level, otherwise yes of course, a lot of differences

Reply:
is this used also for e.g. discovering premixed libraries?